apps/checkout/src/components/Label.tsx

```diff
@@ -1,20 +1,13 @@
-import { Classes } from "@/lib/globalTypes";
-import clsx from "clsx";
-import React, { PropsWithChildren } from "react";
+import {
+  Label as UiKitLabel,
+  LabelProps as UiKitLabelProps,
+} from "@saleor/ui-kit";
+import React from "react";
 
-interface LabelProps extends Classes {
+interface LabelProps extends UiKitLabelProps {
   htmlFor: string;
 }
 
-export const Label: React.FC<PropsWithChildren<LabelProps>> = ({
-  children,
-  htmlFor,
-  className,
-}) => (
-  <label
-    htmlFor={htmlFor}
-    className={clsx("text-text-primary text-base font-regular", className)}
-  >
-    {children}
-  </label>
+export const Label: React.FC<LabelProps> = ({ ...rest }) => (
+  <UiKitLabel {...rest} />
```

Comment:
`htmlFor` is defined explicitly in props but all props are spread, I guess it's wrong

Reply:
The thing here is that we want checkout to be accessible, hence we want to make sure we add this prop to all labels, but not in the ui kit which is to be used also in storefronts, so there we don't want to enforce that practice